### DISCUSSION TITLE
Appinstaller and scale-specific asset packages - question

### DISCUSSION THREAD

**Discussion by benstevens48** [microsoft/WindowsAppSDK#1690] [Q&A] [answered]
This is a simple question about what happens if I install an msixbundle from an appinstaller file on the web rather than the Microsoft Store. The msixbundle has several asset packages for different scales (from 100 to 400) that were automatically created as separate packages in the bundle by Visual Studio. My question is what happens when the display scale changes after the initial install, so that a different scale-specific package becomes more appropriate. Will Windows automatically download and install the new scale-specific package? If so is that immediately or in a few hours time etc and does it require the appinstaller to specify automatic updates? Does it work the same for both regular and flat bundles? I could probably do a test but it would be quite time consuming and I don't know exactly how to confirm it has installed the new scale-specific package, so I was hoping someone in the know could answer here.

**Comment by btueffers** [comment]
Hi @benstevens48, this is a fantastic question! You'll find more direct help if you go to the [MSIX Packaging repo](https://github.com/microsoft/msix-packaging/issues).

**Comment by benstevens48** [comment, marked as answer]
For anyone interested, the answer is here - https://github.com/microsoft/msix-packaging/issues/489. Basically the new scale package will be downloaded when the app is next updated, whenever that is.
